src/homepage.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-restricted-syntax */
+
+import Meal from './meal.js';
+
+export function updateMealsNumber(mealsList) {
+  document.getElementById('meals-num').textContent = mealsList.length;
+}
+
+function createMealsObject(mealsList, likesList) {
+  for (const meal of mealsList) {
+    let likes;
+    const likeObj = likesList.filter((obj) => obj.item_id === meal.idMeal);
+    if (likeObj.length === 1) {
+      likes = likeObj[0].likes;
+    } else {
+      likes = 0;
+    }
+    const mealObj = new Meal(
+      meal.strMeal,
+      meal.strMealThumb,
+      meal.idMeal,
+      likes,
+    );
+    mealObj.drawTheMeal();
+  }
+}
+
+async function egyptianMeals() {
```

Comment:
- [ ] It looks like here we could use ES6 best practices (in this case arrow function). 🐱